sql_generators/experiment_monitoring/templates/experiments_daily_active_clients_v1/query.sql

```diff
@@ -13,6 +13,18 @@ WITH
       CROSS JOIN
         UNNEST(experiments) AS e
     )
+  {% elif app_dataset == "monitor_cirrus" %}
+    {{ app_dataset }} AS (
+      SELECT DISTINCT
+        DATE(submission_timestamp) AS submission_date,
+        e.key AS experiment_id,
+        e.value.branch AS branch,
+        client_info.client_id
+      FROM
+        `moz-fx-data-shared-prod.{{ app_dataset }}.enrollment`
```

Comment:
I think this should be user_id, which you'll have to get out of the event extras with a couple changes:
```suggestion
        mozfun.map.get_key(enrollment.extra, "user_id") AS user_id
      FROM
        `moz-fx-data-shared-prod.{{ app_dataset }}.enrollment` AS enrollment
```

Reply:
Will also need to re-generate after the change.